src/Header.js

```diff
@@ -1,13 +1,11 @@
 import { Component } from 'react';
-// import Fruit from './Fruit.js';
 
 class Header extends Component {
   render() {
     return (
-      <>
-
-
-      </>
+      <footer>
+        <h1>Horned Animals</h1>
+      </footer>
```

Comment:
Good work on lab 1 so far!

On line 6 you used footer tag but we are in the `Header` component. 
In this case, you just need to add an `h1` tag since the Header component functions as the `Header`.

Ex:

```
<div>
    <h1>Horned Beasts</h1>
</div>
```

or use a fragment:

```
<>
 <h1>title </h1>
</>
```